### DISCUSSION TITLE
Scrolling through the documentation

### DISCUSSION THREAD

**Discussion by directormac** [LazyVim/LazyVim#819] [Q&A] [answered]
![image](https://github.com/LazyVim/LazyVim/assets/5866196/eeab0a50-0626-4918-bac8-a6e148153780)

Only mouse scroll works, <C-n> does not or <C-p> how do i bind them?

**Comment by dpetka2001** [comment]
If you press `K` a second time your cursor will go in the documentation and you can use regular vim navigation.

**Comment by nipuna-perera** [comment]
After going into the doc, is there a key to get out of it? I usually have
to do `<space>wd`

**Reply by dpetka2001** [reply]
Just press `q`.

**Comment by folke** [comment, marked as answer]
You can scroll through lsp docs with `<c-f>` & `<c-b>`. No need to make the doc window active

**Reply by directormac** [reply]
Thank you!!! Really appreciate what you are doing Making newbies like me super lazy.

**Comment by nipuna-perera** [comment]
Pressing `q` somehow didn't work earlier when I tried. But now it does.
That is sufficient for me.
